exonum-java-binding/core/rust/build.rs

```diff
@@ -5,12 +5,14 @@ use exonum_build::ProtobufGenerator;
 use std::env::var_os;
 
 fn main() {
-    // We need to link to libc++.dylib on Mac if using static linkage with RocksDB
-    // This is because `librocksdb.a` provided with Homebrew package depends on
-    // `libc++.dylib` and in case of static linkage the resulting `libjava_bindigs.dylib`
-    // also needs to be linked against `libc++.dylib`.
     if cfg!(target_os = "macos") && var_os("ROCKSDB_STATIC").is_some() {
+        // We need to link to libc++.dylib on Mac if using static linkage with RocksDB
+        // This is because `librocksdb.a` provided with Homebrew package depends on
+        // `libc++.dylib` and in case of static linkage the resulting `libjava_bindigs.dylib`
+        // also needs to be linked against `libc++.dylib`.
```

Comment:
Do we have anywhere in the installation instructions 'install libc++ of a compatible version'? Or is it available on Mac by default?

Reply:
It is available by default. You don't need to install zstd either, because it is dependency of rocksdb package